.github/workflows/push-tags.yaml

```
@@ -0,0 +1,66 @@
+name: Push Tag Build
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version (e.g. v0.1.0)'
+        required: true
+      admin:
+        description: 'Please authorise yourself'
+        required: true
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+jobs:
+  push-tag-image-to-container-registry:
+    runs-on: ubuntu-18.04
+    if: {{ github.repository == 'rook/rook' && github.admin == github.event.inputs.admin }}
```

here, we need to set admin var in git secretes but we can also use something like
```suggestion
    if: {{ github.repository == 'rook/rook' && github.actor == github.event.inputs.admin }}
```
github.actor is defined var in action(the person who is triggering the action)